### PR TITLE
read dl2 and sim info into pyirf internal format

### PR DESCRIPTION
**Pull request by vuillaut**
I will add a unit test

### CODE REVIEW

File: lstchain/io/io.py [hunk 1]
```diff
@@ -18,7 +18,9 @@
 from astropy.utils import deprecated
 from ctapipe.instrument import OpticsDescription, CameraGeometry, CameraDescription, CameraReadout, \
     TelescopeDescription, SubarrayDescription
-
+from pyirf.simulations import SimulatedEventsInfo
+from astropy import table
+import pandas as pd
 
 __all__ = ['read_simu_info_hdf5',
            'read_simu_info_merged_hdf5',
```
Comment (chaimain) [inline lstchain/io/io.py:26]: Here 'read_dl2_to_pyirf' should be included as well

Comment (chaimain) [reply lstchain/io/io.py:26]: And also in the init script

Comment (vuillaut) [reply lstchain/io/io.py:26]: Done

Comment (vuillaut) [reply lstchain/io/io.py:26]: (in init it's only necessary to expose the function in `lstchain.io` which is useful for external users - I don't think it's necessary here)

Comment (chaimain) [reply lstchain/io/io.py:26]: Oh ok! Understood, thanks!

**Review by JouvinLea — APPROVED** [review]

**Comment by MaxNoe** [comment]
Maybe working on a pyirf branch until everything is ready might be a good idea instead of merging pieces into the master?

**Comment by rlopezcoto** [comment]
I agree with @MaxNoe, specially now that we are finalizing the tests on the data before releasing v0.6.1, it would be nice not to include in the master any non-necessary changes not to break any compatibilities and have reproducible results

**Comment by moralejo** [comment]
For me tests are not passing, it complains about missing pyirf.

**Comment by chaimain** [comment]
Yes, the setup file also has to be updated, and as @MaxNoe suggested, we will work on a separate branch to work on our scripts, until both lstchain and pyirf are ready for the DL3 scripts.

**Comment by moralejo** [comment]
Ok, please revert these changes. Amazingly I cannot even run the R0 to DL1 step - though it should not at all be related to IRFs, I guess

**Comment by moralejo** [comment]
Actually it is a bit urgent, would be nice if sb can fix this today

**Comment by rlopezcoto** [comment]
@moralejo I guess that you need to update the environment with:
`conda env update -n lst-dev -f environment.yml`

does that work?

**Comment by moralejo** [comment]
> @moralejo I guess that you need to update the environment with:
> `conda env update -n lst-dev -f environment.yml`
> 
> does that work?

Yes, thanks! I thought pip install would do it...

**Pull request merged**